<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapRangeLargeForm</class>
 <widget class="QWidget" name="MapRangeLargeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lbl_south">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>southBoundMap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_north">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>northBoundMap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_west">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>westBoundMap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_east">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>eastBoundMap</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QDoubleSpinBox" name="tet_southBoundMap">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>98000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>7000.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="tet_northBoundMap">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>14000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>98000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>7000.000000000000000</double>
           </property>
           <property name="value">
            <double>14000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="tet_westBoundMap">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>98000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>7000.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="tet_eastBoundMap">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>14000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>98000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>5000.000000000000000</double>
           </property>
           <property name="value">
            <double>14000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
